### PR TITLE
Last : Fix multiple fasta

### PR DESCRIPTION
**Pull request by RomainDallet**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/last/lastal.xml [hunk 1]
```diff
@@ -75,15 +75,14 @@
 
     <inputs>
         <expand macro="input_db"/>
-        <param name="query_fasta" type="data" format="FASTA" label="Queries fasta file"/>
```
Comment (bernt-matthias) [inline tools/last/lastal.xml:78]: Just `label="Query sequences"`

Comment (bernt-matthias) [reply tools/last/lastal.xml:78]: I would like to see a test for at least one other output format, preferably fastq-sanger.

File: tools/last/lastal.xml [hunk 2]
```diff
@@ -75,15 +75,14 @@
...
-                <option value="0" selected="true">fasta or fastq-ignore</option>
+            <param argument="-Q" type="select" multiple="false" label="Query input format">
+                <option value="0" selected="true">FASTA</option>
                 <option value="1">fastq-sanger</option>
                 <option value="2">fastq-solexa</option>
                 <option value="3">fastq-illumina</option>
-                <option value="4">prb</option>
                 <option value="5">PSSM</option>
             </param>            
             <param argument="-f" type="select" multiple="false" label="Output format">
```
Comment (bernt-matthias) [inline tools/last/lastal.xml:88]: The output format selection is not reflected in the `<output>` section.

File: tools/last/tool_data_table_conf.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<tables>
```
Comment (bernt-matthias) [inline tools/last/tool_data_table_conf.xml:1]: Is this data table used in a test?

File: tools/last/lastal.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/last/lastal.xml:78]: If possible I would prefer that galaxy sets the format for fastx. You can get it from `$query_fasta.ext`

File: tools/last/tool_data_table_conf.xml (hunk 3 above)
Comment (RomainDallet) [reply tools/last/tool_data_table_conf.xml:1]: Yes, in the 2 tests of lastal. The values from the parameter "database" come from this table.

Comment (bernt-matthias) [reply tools/last/tool_data_table_conf.xml:1]: I see. Wondering if the case `lastdb` works, since no format is set there.

Comment (bgruening) [reply tools/last/tool_data_table_conf.xml:1]: @RomainDallet but this file is not needed, isn't it. You have the tool_data_table_conf.xml.smple file already included.

Comment (RomainDallet) [reply tools/last/tool_data_table_conf.xml:1]: Oh yes, i'll remove it!

**Comment by bgruening** [comment]
@RomainDallet there is now a merge conflict. Can you resolve that please or can I help here?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@RomainDallet thanks a lot!

**Pull request merged**